src/server.c

```diff
@@ -5564,23 +5572,25 @@ int redisFork(int purpose) {
             server.child_pid = childpid;
             server.child_type = purpose;
             server.stat_current_cow_bytes = 0;
+            server.stat_current_processed_keys = 0;
+            server.stat_keys_on_bgsave_start = 0;
```

Comment:
we can sample `dbTotalServerKeyCount` here, and then we don't need to do that for in aof.c and the two places in rdb.c